registration/views.py

```diff
@@ -403,9 +419,11 @@ def render_done(self, form, **kwargs):
         self.storage.reset()
         # also clean out the session
         clean_session(self.request.session)
-        clean_cobrand_session(self.request.session)
-        clean_widget_session(self.request.session)
-        clean_email_source_session(self.request.session)
+        # I Don't think we need to clear those session values on registration done, as it is going to be cleared on the
+        # next visit for Start page anyways.
+        # clean_cobrand_session(self.request.session)
+        # clean_widget_session(self.request.session)
```

Comment:
@waseem-omar ok. I can't think of any other situations other than removing the cleaning of sessions at the beginning. can you put in a view test to make sure cleaning at start happens?